db/migrate/20251126005628_enable_variant_tag_and_enable_inventory_for_enterprise_using_it.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class EnableVariantTagAndEnableInventoryForEnterpriseUsingIt < ActiveRecord::Migration[7.1]
+  def up
+    Flipper.disable_group(:variant_tag, "enterprise_created_after_2025_08_11")
+    Flipper.disable_group(:variant_tag, "old_enterprise_with_no_inventory")
```

Comment:
I think this group `old_enterprise_with_no_inventory` doesn't exist, it was only ever part of this PR. I'm not sure if this will cause an error, but I guess we'll find out on staging.